changelog/2926.trivial.rst

```diff
@@ -0,0 +1,8 @@
+User can access the source ID of HelioviewerClient using fewer keywords. Either by `observatory` and `measurement` or `instrument` and `measurement` as this much information is enough to get the source ID for most of the cases.
+
+For example if we want to use `sunpy.net.helioviewer.HelioviewerClient.download_jp2` function we can use it in the following ways:
+1. filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', instrument='HMI', detector=None, measurement='continuum')
+2. filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', measurement='continuum')
+3. filepath = hv.download_jp2('2012/07/03 14:30:00', instrument='HMI', measurement='continuum')
+
+All the above three ways will give the same result. An error message is thrown if we get more than one value for a partial key.
```

Comment:
The rendered version looks like your missing whitespace: 

![screenshot from 2019-02-08 12-22-40](https://user-images.githubusercontent.com/1392107/52478130-48ee9a80-2b9c-11e9-808a-3ff866c643bc.png)

Reply:
But I gave proper spaces while writing that. Do I need to build the docs to see how it looks after rendering?

Reply:
Yes or have a look at the giles link in the checks section.

Reply:
Ok